src/app.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import Header from './header';
+
+import JudgingRoundDisplay from './judging_round_display';
+import JudgingRoundSelector from './judging_round_selector';
+
+
+class App extends React.Component {
+    constructor(props) {
+	super(props);
+	this.state = {
+            judging_round: null,
+	};
+    }
+
+    select_judging_round = (_, data) => {
+	this.setState({judging_round: data.value})
+    }
+
+    render() {
+	return (<div>
+		<Header value='Application Allocator Setup'/>
```

Comment:
I'm not going to leave comments on style (apart from this one) but I will say that we've been using prettier on the directory and it's been a complete joy not having to make styling decisions on our code.

Reply:
I see if I can get that hooked up with emacs.  Emacs has a JavaScript mode which is fairly opinionated and I'm not sure how well the two will play with each other.  Not going to fix for this PR though.